README-idview.md

```diff
@@ -0,0 +1,130 @@
+Idview module
+============
+
+Description
+-----------
+
+The idview module allows to ensure presence and absence of idviews and idview host members.
+
```

Comment:
It would be nice to have a "Use cases" section (as in `ipaidrange`), and include the differences in behavior from the CLI commands.